app/assets/javascripts/search.js

```diff
@@ -1,114 +1,121 @@
+      var searchResultData = { 'urls': [] }
+
+      var searchURLs = search.extractSearchURLs($searchResults)
+
+      var searchSuggestion = search.extractSearchSuggestion()
 
       if (searchURLs.length) {
-        searchResultData['urls'] = searchURLs.toArray();
+        searchResultData['urls'] = searchURLs.toArray()
       }
 
       if (searchSuggestion !== null) {
-        searchResultData['suggestion'] = searchSuggestion;
+        searchResultData['suggestion'] = searchSuggestion
       }
 
-      return searchResultData;
+      return searchResultData
     },
     enableLiveSearchCheckbox: function ($searchResults) {
-      GOVUK.liveSiteSearch.init();
+      GOVUK.liveSiteSearch.init()
     },
     extractSearchSuggestion: function () {
-      var $suggestion = $('.spelling-suggestion a');
+      var $suggestion = $('.spelling-suggestion a')
 
       if ($suggestion.length) {
-        return $suggestion.text();
+        return $suggestion.text()
       } else {
-        return null;
+        return null
       }
     },
     extractSearchURLs: function ($searchResults) {
       if ($searchResults.length <= 0) {
-        return [];
+        return []
       }
 
-      function extractSearchURL(index, element) {
-        var foundURL = $(element).find('h3 a');
+      function extractSearchURL (index, element) {
+        var foundURL = $(element).find('h3 a')
 
         if (foundURL.parents('.descoped-results').length) {
-          return $.makeArray(foundURL.map(function(index, item) {
+          return $.makeArray(foundURL.map(function (index, item) {
             return {
               href: $(item).attr('href'),
               descoped: true
-            };
-          }));
+            }
+          }))
         } else {
           return {
             href: foundURL.attr('href')
-          };
+          }
         }
       }
 
-      return $searchResults.children().map(extractSearchURL);
+      return $searchResults.children().map(extractSearchURL)
     },
     trackSearchClicks: function ($searchResults) {
-      if($searchResults.length === 0 || !GOVUK.cookie){
-        return;
+      if ($searchResults.length === 0 || !GOVUK.cookie) {
+        return
       }
 
-      $searchResults.on('click', 'a', function(e){
-        var $link = $(e.target),
-            sublink = '',
-            position, href, startAt;
+      $searchResults.on('click', 'a', function (e) {
+        var $link = $(e.target)
+
+        var sublink = ''
+
+        var position; var href; var startAt
 
-        var getStartAtValue = function() {
-          var queryString = window.location.search,
-              startAtRegex = /(&|\?)start=([0-9]+)(&|$)/,
-              startAt = 0,
-              matches;
+        var getStartAtValue = function () {
+          var queryString = window.location.search
 
-          matches = queryString.match(startAtRegex);
+          var startAtRegex = /(&|\?)start=([0-9]+)(&|$)/
+
```

Comment:
Nitpick: we probably don't need a newline between each variable declaration.

Reply:
See https://github.com/alphagov/finder-frontend/pull/1086#discussion_r280335766